Makefile

```diff
@@ -272,20 +276,31 @@ get-ci-tools:
 
 # Run go fmt against code
 gofmt: get-ci-tools
-	$(CI_TOOLS_REPO_DIR)/test-runner/gofmt.sh
+	GOWORK=$(GOWORK) $(CI_TOOLS_REPO_DIR)/test-runner/gofmt.sh
+	GOWORK=$(GOWORK) $(CI_TOOLS_REPO_DIR)/test-runner/gofmt.sh ./api
 
 # Run go vet against code
 govet: get-ci-tools
-	$(CI_TOOLS_REPO_DIR)/test-runner/govet.sh
+	GOWORK=$(GOWORK) $(CI_TOOLS_REPO_DIR)/test-runner/govet.sh
```

Comment:
@gibizer @stuggi  I already set this here but somehow is failing [1].
I assumed this is related to the fact we added `go.work` within the repo, so I explicitly added the `GOWORK` var within the script [2]. 
By doing this we can remove `GOWORK` here (it can be applied to all the other operators), not sure if you have preferences or this approach has some drawbacks I'm overlooking.

[1] https://github.com/openstack-k8s-operators/cinder-operator/actions/runs/3573061209/jobs/6006679452
[2] https://github.com/openstack-k8s-operators/openstack-k8s-operators-ci/pull/56/files

Reply:
Ahh I always forget that our workflow are not calling make targets (they should I think). So you have to add it to the workflows too https://github.com/openstack-k8s-operators/cinder-operator/blob/6240496cc613a1f7843c0960158210d2e9203631/.github/workflows/golangci-lint.yaml#L21-L26

Reply:
... or, according to the review [1], `GOWORK=off` is automatically pushed in the script, preserving the existing behavior, and I just have to remove the  `GOWORK` references in the `Makefile` here, am I right?
@gibizer added you to [1] 

[1] https://github.com/openstack-k8s-operators/openstack-k8s-operators-ci/pull/56

Reply:
Yes, that would work too.